Userland/Utilities/test-bencode.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021, Cesar Torres <shortanemoia@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <LibBencode/Bencode.h>
+
+constinit StringView decode_result_3 { "( (announce:  7564703a2f2f747261636b65722e746f74616c6c796e6f74636f707065727375726665722e746b3a363936392f616e6e6f756e6365,\n"
+                                       " )\n"
+                                       " (info:  (   (name:    73616d706c655f66696c652e66696c65,\n"
+                                       "   )\n"
+                                       "   (piece length:    16384,\n"
+                                       "   )\n"
+                                       "   (pieces:    000102030405060708090a0b0c0d0e0f10111213,\n"
+                                       "   )\n"
+                                       "   (length:    420,\n"
+                                       "   )\n"
+                                       " )\n"
+                                       " (dummy:  (\n"
+                                       "   6974656d31,\n"
+                                       "   2,\n"
+                                       "   6974656d33,\n"
+                                       "  )\n"
+                                       " )\n" };
+
+void print_node(int indent_level, StringBuilder& sb, Bencode::BencodeNode const& node);
```

Comment:
Since `indent_level` should be non-negative, let's use a unsigned type here, such as `size_t`.